src/time-picker/time-picker.jsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import WindowListenable from '../mixins/window-listenable';
```

Comment:
This one confused me a little. It looks like `time-picker.jsx` had the mixin but was referencing a function that wasn't defined on the class. I'm assuming that's the equivalent of a noop so I just removed the mixin without adding the `EventListener`